app/src/main/java/com/example/parkingsystem/mvp/view/ParkingView.java

```diff
@@ -38,7 +38,9 @@ public void showDialog(ListenerDialogFragment listenerDialogFragment) {
     public void showPopUp(int spaces) {
         Context context = getContext();
         if (context != null) {
-            Snackbar.make(binding.getRoot(), context.getResources().getString(R.string.snack_bar_button_click_message_main_activity, String.valueOf(spaces)), Snackbar.LENGTH_SHORT).show();
+            Snackbar.make(binding.getRoot(),
+                    context.getResources().getString(R.string.snack_bar_button_click_message_main_activity, String.valueOf(spaces)),
+                    Snackbar.LENGTH_SHORT).show();
```

Comment:
pls use a generic method to modularize the behaviour because you have the same code bellow

Reply:
fixed